themes/openy_themes/openy_rose/openy_rose.theme

```diff
@@ -409,3 +409,16 @@ function openy_rose_theme_suggestions_form_alter(array &$suggestions, array $var
     array_push($suggestions, 'form__' . $variables['element']['#form_id']);
   }
 }
+
+/**
+ * Preprocess paragraph - social list.
+ *
+ * @param $variables
+ */
+function openy_rose_preprocess_paragraph_social_list(&$variables) {
+  $block = $variables['content']['field_prgrf_sl_block'];
+  if (isset($block[0]['#name']) && isset($block[0]['#display_id'])) {
```

Comment:
what happens when there are 2+ blocks?

Reply:
We have 2 blocks as expected. I checked.